### PR TITLE
French translation (locale fr)

### PR DESCRIPTION
**Pull request by Spawin**
in camera_picker_text_delegate.dart

### CODE REVIEW

File: lib/src/delegates/camera_picker_text_delegate.dart [hunk 1]
```diff
@@ -177,3 +178,76 @@ class EnglishCameraPickerTextDelegate extends CameraPickerTextDelegate {
   String sSwitchCameraLensDirectionLabel(CameraLensDirection value) =>
       'Switch to the ${sCameraLensDirectionLabel(value)} camera';
 }
+
+/// Text delegate implements with French.
```
Comment (AlexV525) [inline lib/src/delegates/camera_picker_text_delegate.dart:182]: You'll need a French explanation here.

**Review by AlexV525 — COMMENTED** [review]
Thanks for the contribution. I have only one nit-picking here, and also a recording that using Voice-Over or Talkback with the text delegates is required to verify it they are proper for semantics.

**Comment by AlexV525** [comment]
Are you still working on this PR?

**Comment by Spawin** [comment]
> Are you still working on this PR?

I made the requested changes but I don't think I can do the recording with Voice-Over. So I preferred to close the pull request to avoid clutter.

**Comment by AlexV525** [comment]
I can do a recording if you have trouble with it, a Voice-Over or a Talkback recording can both be valid.

**Comment by AlexV525** [comment]
The recording is to make sure that translated words behave normally when they're being played.

**Comment by Spawin** [comment]
> I can do a recording if you have trouble with it, a Voice-Over or a Talkback recording can both be valid.

Ok thanks. 


> The recording is to make sure that translated words behave normally when they're being played.

Alright I understand. I will reopen the PR and also make some changes because fields have been added to the recent update.

**Pull request closed without merging**